docs/pages/server-access/guides/openssh.mdx

```diff
@@ -33,230 +33,27 @@ and running, but in the long run, we would recommend replacing `sshd` with `tele
-$ tctl get node/openssh-node --format text
-```
-
-If only one node is displayed and you have `jq` installed, you can run the
-following command to get your node's UUID:
-
-```code
-$ tctl get node/openssh-node --format=json | jq -r ".[0].metadata.name"
-```
-
-Otherwise, find your node's UUID in the `metadata.name` field of the YAML
-output of this command:
-
-```code
-$ tctl get node/openssh-node
-```
-
-</Details>
-
-#### Create the host certificate
-
-When creating host certificates, it is important to specify all the domain names
-and addresses that refer to your node. If you try to connect to a node with a 
-name or address that was not specified when creating it's host certificate,
-Teleport will reject the SSH connection.
-
-On your local machine, assign the IP address, fully qualified domain name of
-your node, and the node's UUID to an environment variable. If you won't be
-connecting to your node with its hostname, you can safely omit it.
-
-```code
-$ ADDR=1.2.3.4,openssh-node,a100fdd0-52db-4eca-a7ab-c3afa7a1564a
-```
-
-Run the following `tctl` command to generate a host certificate:
+On the machine the cluster's Proxy is running on, use the `tctl` tool to generate a new token. In the
+following example, a new token is created with a TTL of five minutes:
 
 ```code
-$ tctl auth sign \
-      --host=${ADDR?} \
-      --format=openssh \
-      --out=myhost
-
-# The credentials have been written to myhost, myhost-cert.pub
+$ tctl tokens add --ttl=5m --type=node --format=text
+(=presets.tokens.first=)
 ```
 
-The above command will result in a private key and certificate.
-
-<Details title="Issuing certificates for multiple hosts">
-
-To generate certificates for multiple hosts, assign the `host` flag to a
-comma-separated list of addresses. Certificates for wildcard domains are not
-supported by OpenSSH, so each domain must be fully qualified.
-
-</Details>
-
-Use `ssh-keygen` to verify the contents of the certificate:
+Copy the `teleport` binary to your `sshd` host. Configure `sshd` and create a node
+resource on your cluster with the command:
 
 ```code
-$ ssh-keygen -L -f myhost-cert.pub
-```
-
-The `Principals` section should contain the address you assigned to `ADDR`
-earlier:
-
-```
-myhost-cert.pub:
-        Type: ssh-rsa-cert-v01@openssh.com host certificate
-        Public key: RSA-CERT SHA256:nHkp6SnrAW4AV00VUaqPgR6SgdyvV9MmjUrYnwZ779A
-        Signing CA: RSA SHA256:euqx2Y8Pq+r0c94GKVNXAklBVTmAJtaQUn3/ehrfEJE (using rsa-sha2-512)
-        Key ID: ""
-        Serial: 0
-        Valid: after 2022-04-22T11:14:16
-        Principals: 
-                1.2.3.4
-                openssh-node
-                a100fdd0-52db-4eca-a7ab-c3afa7a1564a
-        Critical Options: (none)
-        Extensions: 
-                x-teleport-authority UNKNOWN OPTION (len 33)
-                x-teleport-role UNKNOWN OPTION (len 8)
+$ teleport join openssh --address <NODE-ADDRESS> --proxy-server <PROXY-ADDRESS> --join-method token --token <TOKEN>
```

Comment:
I think we should use the `Var` component instead of placeholder text so this is easier to copy/paste.

Also, what parts should the `NODE-ADDRESS` and `PROXY-ADDRESS` include? Scheme, host, and port?